diagnostic.md

```diff
@@ -5,20 +5,21 @@ Record your responses inside the fenced code blocks below each question.
 1.  Give an example of a visual hierarchy that could be modeled with components.
 
     ```md
-    <!-- your response here -->
+    you could have a list that has a component of items.
```

Comment:
yes.  Other examples could be a navbar w/ buttons, or the sidebar of
a blog w/ comments.